lbrynet/core/Wallet.py

```diff
@@ -946,6 +951,9 @@ def claim_name(self, name, bid, metadata, certificate_id=None, claim_address=Non
             raise Exception(msg)
 
         claim = self._process_claim_out(claim)
+
+        # lbry/issue#958
+        claim['permanent_url'] = str(name) + "#" + claim["claim_id"]
```

Comment:
Hmm... is there a way to refactor all this so that the same code isn't duplicated three times?

In either case, should be using `.format()` like the others.